### PR TITLE
Add api document for v2 membership service

### PR DESCRIPTION
**Pull request by babygoat**
This patch adds the api document for the version 2 membership service
endpoints including both email login and oauth login.

### CODE REVIEW

**Comment by babygoat** [comment]
@nickhsine

File: doc/oauth.apib [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+## Google oauth request [/v2/auth/google{?destination}]
+Redirect a user request to google oauth server
+
+### Redirect google request [GET]
++ Parameters
+    + destination: https://www.twreporter.org
+
++ Response 302
+
+## Google oauth response [/v2/auth/google/callback]
```
Comment (nickhsine) [inline doc/oauth.apib]: Add query params(`state` and `code`) here?

Comment (babygoat) [reply doc/oauth.apib]: done

File: doc/auth.apib [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
++ Parameters
+    + email: user@example.com (required)
+    + token: 26dlFidiTVY= (required)
+    + destination: https://www.twreporter.org (optional)
+
++ Response 302
+    
+## Token [/v2/token]
+Authenticate user request and grant access token to the corresponding domain
+
```
[the remark this answers is not in the document]
Comment (babygoat) [reply doc/auth.apib]: done

File: doc/auth.apib [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+## Activate [/v2/activate{?email,token,destination}]
+Send identity token if valid user sigins
+
+### Activate user [GET]
++ Parameters
+    + email: user@example.com (required)
+    + token: 26dlFidiTVY= (required)
+    + destination: https://www.twreporter.org (optional)
+
++ Response 302
```
[the remark this answers is not in the document]
Comment (babygoat) [reply doc/auth.apib]: done

**Comment by babygoat** [comment]
updated.

**Comment by nickhsine** [comment]
LGTM

**Pull request merged**